indexing-service/src/test/java/org/apache/druid/indexing/common/task/CompactionTaskRunTest.java

```diff
@@ -191,6 +192,10 @@ public void testRun() throws Exception
           Intervals.of("2014-01-01T0%d:00:00/2014-01-01T0%d:00:00", i, i + 1),
           segments.get(i).getInterval()
       );
+      Assert.assertEquals(
+          new DynamicPartitionsSpec(5000000, Long.MAX_VALUE),
```

Comment:
Perhaps save this as a named constant since it's used a lot?

Reply:
Added.